client/src/components/Draftail/CommentableEditor/CommentableEditor.test.tsx

```diff
@@ -139,18 +139,15 @@ describe('CommentableEditor', () => {
   it('has control', () => {
     commentApp.setVisible(true);
     const editor = mount(getEditorComponent(commentApp));
-    const controls = editor.findWhere(
-      (n) => n.name() === 'ToolbarButton' && n.prop('name') === 'comment',
-    );
+    const controls = editor.find('DraftailEditor').prop('controls');
     expect(controls).toHaveLength(1);
+    expect(controls[0].inline).toBeTruthy();
```

Comment:
This is now testing the props rather than the rendered buttons, as the buttons are only conditionally rendered in a toolbar. I think that’s also appropriate as CommentableEditor is only responsible for setting the props correctly anyway.

Reply:
Sounds like a good plan!